test/spec/server-discovery-and-monitoring/rs/secondary_ignore_ok_0.json

```diff
@@ -1,5 +1,5 @@
 {
-  "description": "New primary",
+  "description": "Secondary ignored when ok is zero",
```

Comment:
It's the same commit that the setVersion/electionId landed in, is there something we should do to further connect it?

https://github.com/mongodb/specifications/commit/5bd06a81d97850a1365d73f08bbb7dea562e72b9

Reply:
it just might be confusing that this is a partial spec sync; like when we look at the main PR, we'd likely compare files, and then someone might have a question about why these ones are missing; I think it's ok to include them in this PR, but you should update *this* PR's description to say that it includes a partial sync of that commit, and then update the *other* PR description to say that this specific test was synced in this linked PR, so that other reviewers won't have to go looking for them

Reply:
done!